ext/afform/core/ang/af/afRecaptcha.component.js

```diff
@@ -0,0 +1,8 @@
+(function(angular, $, _) {
+  angular.module('af').component('afRecaptcha', {
+    templateUrl: '~/af/afRecaptcha.html',
+    controller: function($element) {
+      $('div.g-recaptcha').attr('data-sitekey', $($element).attr('recaptchakey'));
```

Comment:
@kurund this is "the jQuery way" of doing things, where you manipulate the dom directly. To do this the angular way, you can keep the MetadataInjector code the same (where it sets the attribute `recaptchakey`) but you can use Angular to read it. Like this:
```suggestion
    bindings: {
      recaptchakey: '@',
    },
    controller: function($element) {
```